### ISSUE TITLE
Query Builder - Queries against Saved Questions with Applied Filters, Filter disappears once Saved

### ISSUE THREAD

**Issue by chucksense** [metabase/metabase#10425]
**Describe the bug**
Query Builder - Queries against Saved Questions with Applied Filters, the filter disappears once Saved.

**To Reproduce**
This applies to other database types, but to cleanly reproduce it with the sample dataset, do the following:

1. Create a question with a SQL query as follows:

```sql
WITH tmp_user_order_dates as (
  SELECT 
    o.USER_ID,
    o.CREATED_AT,
    o.QUANTITY
  FROM 
    ORDERS o
),

tmp_prior_orders_by_date as (
  select
      tbod.USER_ID,
      tbod.CREATED_AT,
      tbod.QUANTITY,
      (select count(*) from tmp_user_order_dates tbod2 where tbod2.USER_ID = tbod.USER_ID and tbod2.CREATED_AT < tbod.CREATED_AT ) as PRIOR_ORDERS
  from tmp_user_order_dates tbod
)

select
  date_trunc('day', tpobd.CREATED_AT) as "Date",
  case when tpobd.PRIOR_ORDERS > 0 then 'Return' else 'New' end as "Customer Type",
  sum(QUANTITY) as "Items Sold"
from tmp_prior_orders_by_date tpobd
group by date_trunc('day', tpobd.CREATED_AT), "Customer Type"
order by date_trunc('day', tpobd.CREATED_AT) asc
```

2. Create a second question referencing the first saved question.  In this, apply a Group by `Date : Week` and `Customer Type` (this group by isn't necessary to reproduce, but is how my screenshot is structured so including it).  Apply a filter on `Date` e.g., `Previous 30 Days`.  Run the query.  Save the question.  As soon as the question is saved, and any time you revisit the question, the Filter text "disappears" from the UI and you can't click into it to modify it.  You can only click the `X` to remove it.

![Monosnap 2019-07-29 09-47-21](https://user-images.githubusercontent.com/199588/62053586-ea6ab680-b1e5-11e9-9f0d-fa595d30e669.jpg)

**Expected behavior**
The filter should show like it does prior to the save:

![Monosnap 2019-07-29 09-48-05](https://user-images.githubusercontent.com/199588/62053609-f9516900-b1e5-11e9-8bd8-3ecbf8b73a7d.jpg)

**Information about your Metabase Installation:**
- Your browser and the version: Cross-browser issue (tested Chrome, Safari, and Native Metabase macOS app)
- Your operating system: macOS 
- Your databases: Reproducible on both Postgresql and the Sample Dataset
- Metabase version: 0.32.9 (but I saw it in an earlier version too)
- Metabase hosting environment: Reproducible in both macOS app and Elastic Beanstalk instance
- Metabase internal database: Reproducible in both H2 and Postgresql

**Severity**
Very annoying for saved questions as I have to remove and reapply a filter entirely anytime I just want to tweak one.  But not blocking anything.

**Comment by flamber** [comment]
Hi @chucksense 
This looks like a duplicate of #8971. If I remember correctly, it's already fixed on the preview of 0.33

**Comment by chucksense** [comment]
Yep, that looks right. Thanks @flamber!
